kani-compiler/src/codegen_cprover_gotoc/codegen/rvalue.rs

```diff
@@ -493,6 +494,9 @@ impl<'tcx> GotocCtx<'tcx> {
                     "https://github.com/model-checking/kani/issues/541",
                 )
             }
+            Rvalue::CopyForDeref(place) => {
```

Comment:
Might be helpful to refer to the comment in rustc that mentions that at the codegen level, this is just a read: https://github.com/rust-lang/rust/blob/1673f1450eeaf4a5452e086db0fe2ae274a0144f/compiler/rustc_middle/src/mir/syntax.rs#L1055